src/modules/Kdump.rb

```diff
@@ -897,19 +901,26 @@ def proposal_warning
       return {} if free_space.nil? || requested_space.nil?
 
       warning = {}
+      warning_string = ""
 
       if free_space < requested_space
+        # TRANSLATORS: warning message in installation proposal. Do not translate %{requested} and
+        # %{available} - they are replaced with actual sizes later.
+        warning_string = format(_(
+          "Warning! There might not be enough free space to have kdump enabled. " \
+          "%{required} required for saving a kernel dump, but only %{available} are available."
+        ), required: String.FormatSizeWithPrecision(requested_space, 2, true), available: String.FormatSizeWithPrecision(free_space, 2, true))
+      end
+
+      if Bootloader.getLoaderType == "systemd-boot" && @add_crashkernel_param
+        warning_string += "<br>" if !warning_string.empty?
```

Comment:
minor tip: you can use `unless` instead of `if !`

Reply:
There has been some more :-)